pulsar-broker/src/main/java/org/apache/pulsar/broker/service/Topic.java

```diff
@@ -76,6 +76,14 @@ default long getOriginalSequenceId() {
         }
 
         void completed(Exception e, long ledgerId, long entryId);
+
+        default long getLowestSequenceId() {
```

Comment:
I don't see how these sequence ids relate to a `Topic`.

 * What's the lowest sequenceId ? I think this only apply to 1 single batch.
 * Sequence id are *per producer* anyway

Reply:
It is in the inner class named PublishContext of Topic

Reply:
> It is in the inner class named PublishContext of Topic

Which already contains a sequence id field